feature/announcement/src/main/java/io/github/droidkaigi/confsched2022/feature/announcement/Announcements.kt

```diff
@@ -0,0 +1,252 @@
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import dev.icerock.moko.resources.compose.stringResource
+import io.github.droidkaigi.confsched2022.designsystem.components.KaigiScaffold
+import io.github.droidkaigi.confsched2022.designsystem.components.KaigiTopAppBar
+import io.github.droidkaigi.confsched2022.designsystem.theme.KaigiColors
+import io.github.droidkaigi.confsched2022.designsystem.theme.KaigiTheme
+import io.github.droidkaigi.confsched2022.model.AnnouncementsByDate
+import io.github.droidkaigi.confsched2022.model.fakes
+import io.github.droidkaigi.confsched2022.strings.Strings
+import io.github.droidkaigi.confsched2022.ui.UiLoadState.Error
+import io.github.droidkaigi.confsched2022.ui.UiLoadState.Loading
+import io.github.droidkaigi.confsched2022.ui.UiLoadState.Success
+import kotlinx.collections.immutable.PersistentList
+import kotlinx.datetime.Clock
+import kotlinx.datetime.DateTimeUnit
+import kotlinx.datetime.LocalDate
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.minus
+import kotlinx.datetime.todayIn
+
+@Composable
+fun AnnouncementsScreenRoot(
+    viewModel: AnnouncementsViewModel = hiltViewModel(),
+    showNavigationIcon: Boolean = true,
+    onNavigationIconClick: () -> Unit,
+) {
+    val uiModel by viewModel.uiModel
+    Announcements(
+        uiModel = uiModel,
+        showNavigationIcon = showNavigationIcon,
+        onNavigationIconClick = onNavigationIconClick
+    )
+}
+
+@Composable
+fun Announcements(
+    uiModel: AnnouncementsUiModel,
+    showNavigationIcon: Boolean,
+    modifier: Modifier = Modifier,
+    onNavigationIconClick: () -> Unit,
+) {
+    KaigiScaffold(
+        modifier = modifier,
+        topBar = {
+            KaigiTopAppBar(
+                showNavigationIcon = showNavigationIcon,
+                onNavigationIconClick = onNavigationIconClick,
+                title = {
+                    Text(
+                        text = stringResource(Strings.announcement_top_app_bar_title),
+                    )
+                },
+            )
+        },
+    ) { innerPadding ->
+        when (uiModel.state) {
+            is Error -> {
+                uiModel.state.value?.printStackTrace()
+                TODO()
+            }
+            is Success -> {
+                if (uiModel.state.value.isNotEmpty()) {
+                    AnnouncementContentList(
+                        announcements = uiModel.state.value,
+                        innerPadding = innerPadding,
+                    )
+                } else {
+                    EmptyBody()
+                }
```

Comment:
Although not in the design, we have added an "Empty" message when there is no notification.
If the design is completed in the future, it will be necessary to update it.